packages/suite/src/support/messages.ts

```diff
@@ -5746,6 +5812,202 @@ const definedMessages = defineMessages({
         id: 'TR_WAIT_FOR_REBOOT_WEBUSB_DESCRIPTION',
         defaultMessage: 'Please wait for Trezor to reboot and pair the device again.',
     },
+    TR_ONBOARDING_STEP_FIRMWARE: {
+        id: 'TR_ONBOARDING_STEP_FIRMWARE',
+        defaultMessage: 'Firmware',
+    },
+    TR_ONBOARDING_STEP_WALLET: {
+        id: 'TR_ONBOARDING_STEP_WALLET',
+        defaultMessage: 'Wallet',
+    },
+    TR_ONBOARDING_STEP_PIN: {
+        id: 'TR_ONBOARDING_STEP_PIN',
+        defaultMessage: 'PIN',
+    },
+    TR_ONBOARDING_STEP_COINS: {
+        id: 'TR_ONBOARDING_STEP_COINS',
+        defaultMessage: 'Coins',
+    },
+    TR_ONBOARDING_STEP_COINS_LOWERCASE: {
+        id: 'TR_ONBOARDING_STEP_COINS_LOWERCASE',
+        defaultMessage: 'coins',
+    },
+    TR_ONBOARDING_CURRENT_VERSION: {
+        id: 'TR_ONBOARDING_CURRENT_VERSION',
+        defaultMessage: 'Current Version',
+    },
+    TR_ONBOARDING_NEW_VERSION: {
+        id: 'TR_ONBOARDING_NEW_VERSION',
+        defaultMessage: 'New Version',
+    },
+    TR_PIN_MATRIX_DISPLAYED_ON_TREZOR: {
+        id: 'TR_PIN_MATRIX_DISPLAYED_ON_TREZOR',
+        defaultMessage: 'The numbers are displayed on your Trezor',
+    },
+    TR_DEVICE_CONNECTED: {
+        id: 'TR_DEVICE_CONNECTED',
+        defaultMessage: 'Device connected',
+    },
+    TR_DEVICE_CONNECTED_WRONG_STATE: {
+        id: 'TR_DEVICE_CONNECTED_WRONG_STATE',
+        defaultMessage: 'Device detected in incorrect state',
+    },
+    TR_ONBOARDING_DEVICE_CHECK: {
+        id: 'TR_ONBOARDING_DEVICE_CHECK',
+        defaultMessage: 'Security check',
+        description:
+            'Heading for an onboarding step where we ask the user to verify authenticity of his device',
+    },
+    TR_ONBOARDING_DEVICE_CHECK_1: {
+        id: 'TR_ONBOARDING_DEVICE_CHECK_1',
+        defaultMessage: 'My <strong>hologram</strong> looked prtty much incredible',
```

Comment:
Misspelling in "pretty" but I think the whole copy should be updated :smile:

Reply:
it is actually just like it was designed! 😁 Already improved in crowdin, I'll do the sync after rebase